opentech/apply/funds/admin.py

```diff
@@ -74,9 +34,34 @@ class LabAdmin(ModelAdmin):
     menu_label = 'Labs'
 
 
+class NoDeletePermission(PermissionHelper):
+    def user_can_delete_obj(self, user, obj):
+        return False
+
+
 class ApplicationFormAdmin(ModelAdmin):
     model = ApplicationForm
     menu_icon = 'form'
+    list_display = ('name', 'funds', 'rounds', 'labs')
```

Comment:
As discussed, merge funds/rounds/labs into one column "Used by" and indicate with a label what type it is.